### PR TITLE
Remove extra directions for input box

### PR DESCRIPTION
**Pull request by Sweetman**
### What does this PR do?
This removes the extra "Press Enter to ..." showing in InputBoxOptions. This message comes for free with InputBoxOptions already which is why it came up twice.
### What issues does this PR fix or reference?
@W-4285382@

### CODE REVIEW

**Comment by vazexqi** [comment]
LGTM. How many dialogs were impacted by the double entry?

**Comment by Sweetman** [comment]
Force org display and the create commands were impacted since they used the SelectFileName and SelectUsername parameter gatherers

File: package.json [hunk 1]
```diff
@@ -16,12 +16,13 @@
     "compile": "lerna run --stream compile",
     "lint": "lerna run lint",
     "publish": "node scripts/publish.js",
-    "test": "lerna run test --concurrency 1",
+    "test": "lerna run test --concurrency 1 --stream",
```
Comment (Sweetman) [inline package.json]: @guw @GaryFNC @vazexqi I'm adding this flag to prevent travis timing out because of lack of output.

Comment (Sweetman) [reply package.json]: Talked to Nick and holding off on this change for now.

**Comment by chowwinston** [comment]
This is going into the develop branch. I thought we wanted to have this in for tomorrow's release? @vazexqi @Sweetman

**Comment by Sweetman** [comment]
Right, thanks for catching that @chowwinston

**Comment by Sweetman** [comment]
New PR is #145

**Pull request closed without merging**